### PR TITLE
[WIP] Newsite

### PR DESCRIPTION
**Pull request by ecv19**
Left to do

- [x] links on packages
- [x] images for all packages - Serge
- [x] update the banner (images/carousel)
- [x] links for news items -Esau
- [x] update news - Dylan
- [ ] update README.md (installation instructions)
- [x] favicon
- [x] numfocus
- [x] sponsors/funding
- [x] test on different browsers (chrome, ff, safari, edge) 
        - works in all browsers. Only minor issue is that the flip card flickers for a split second on Safari, will try to resolve.

### CODE REVIEW

**Comment by sjsrey** [comment]
I don't think there are links on the news items?

image

**Comment by sjsrey** [comment]
Would be cool to randomize the listing of the devs on the team page. (Feature request).

**Comment by ecv19** [comment]
There aren't yet, there is a function that it's not working. The title of the news item should redirect to the release or whatever the news url is. I am trying to figure out how to do it, I might have to add a new line with a generic message (View more / view release) and add the link to that

**Comment by sjsrey** [comment]
Links for packages are needed

image

Currently I don't go anywhere after turning the card over and clicking the link

**Comment by ecv19** [comment]
links work, but you really have click right on the middle where it says "visit the page", for some reason the whole button is not clickable (bug?).

**Comment by sjsrey** [comment]
> links work, but you really have click right on the middle where it says "visit the page", for some reason the whole button is not clickable (bug?).

I'm not able to get it to work (on linux and firefox). The cards rotate, but clicking on the "visit the page" doesn't work.

**Comment by sjsrey** [comment]
> > links work, but you really have click right on the middle where it says "visit the page", for some reason the whole button is not clickable (bug?).
> 
> I'm not able to get it to work (on linux and firefox). The cards rotate, but clicking on the "visit the page" doesn't work.

Does this look right (after flipping the card):

image

**Comment by ecv19** [comment]
No, I may have to fix the html files so that they are compatible with different browsers and the card links and flip work

**Comment by sjsrey** [comment]
> No, I may have to fix the html files so that they are compatible with different browsers and the card links and flip work

ok, maybe that is why i can't get the link to work.

File: content/news/sdsc_award_arribas-bel.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Dani Arribas-Bel, SDSC Award
+date: 2020-10-23
+description:  summary: Congratulations to Dani Arribas-Bel for being named Spatial Data Scientist of the Year
```
Comment (knaaptime) [inline content/news/sdsc_award_arribas-bel.md:4]: should this have the `summary:`? 

no worries if intentional, just checking

Comment (ecv19) [reply content/news/sdsc_award_arribas-bel.md:4]: no it shouldn't, I miss that when i merged it.

**Comment by knaaptime** [comment]
https://github.com/pysal/segregation/blob/master/docsrc/_static/images/pysal_favicon.ico

**Comment by knaaptime** [comment]
this may solve the car flip issue https://stackoverflow.com/questions/49202655/flipping-a-card-over-with-css-js-firefox-issue

**Comment by ecv19** [comment]
Screen Shot 2020-11-02 at 2 29 12 PM
Does this look right? Am I missing a few sponsors or need to take out some?

**Pull request merged**